gst/nnstreamer/tensor_aggregator/tensor_aggregator.c

```diff
@@ -118,21 +118,31 @@ enum
  */
 #define DEFAULT_CONCAT TRUE
 
+/**
+ * @brief Template caps string for sink pad. 
+ */
+#define CAPS_STRING_SINK GST_TENSOR_CAP_DEFAULT ";" GST_TENSORS_CAP_WITH_NUM ("1")
+
+/**
+ * @brief Template caps string for src pad.
+ */
+#define CAPS_STRING_SRC GST_TENSORS_CAP_WITH_NUM ("1")
```

Comment:
Doesn't this break backward compatibility?

Reply:
Right. Fixed!